webapp/src/modules/bid/selectors.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { AuthorizationStepStatus } from 'decentraland-ui'
+import { LoadingState } from 'decentraland-dapps/dist/modules/loading/reducer'
+import { NFT } from '../nft/types'
+import { RootState } from '../reducer'
+import { placeBidRequest } from './actions'
+import { getBidStatus } from './selectors'
+
+let rootState: RootState
+
+beforeEach(() => {
+  rootState = {
+    bid: {
+      data: {},
+      loading: [] as LoadingState,
+      error: null
+    }
+  } as RootState
+})
+
+describe('when getting bid status', () => {
```

Comment:
It seems that it's missing the article, would you mind adding it?
```suggestion
describe('when getting the bid status', () => {
```